code/controllers/subsystems/atoms.dm

```diff
@@ -96,6 +99,19 @@ SUBSYSTEM_DEF(atoms)
 
 	return qdeleted || QDELING(A)
 
+// override and GetArguments() exists for mod-override/downstream hook functionality.
+// Useful for total-overhaul type modifications.
+/atom/proc/AdjustInitializeArguments(list/arguments)
+	// Lists are passed by reference so can simply modify the arguments list without returning it
+
+/datum/controller/subsystem/atoms/proc/GetArguments(atom/A, list/mapload_arg, created=TRUE)
```

Comment:
Shouldn't this default to just returning `mapload_arg` if `adjust_init_arguments` isn't set? Otherwise you're adding a list copy, a var check, an assoc list lookup and a list op to the init for every atom on Neb for no gain.